package.json

```diff
@@ -52,6 +52,7 @@
   },
   "peerDependencies": {
     "@fortawesome/fontawesome-free": "5.7.2",
+    "color": "^3.1.0",
```

Comment:
You might want to add "styled-components" as well inside your "peerDependencies"